pkg/pending/cache.go

```diff
@@ -0,0 +1,105 @@
+package pending
+
+import (
+	"context"
+	"sort"
+	"sync"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/pkg/errors"
```

Comment:
this package should be aliased as `errs` (as suggested on Slack this week)

Reply:
done 640536aa